src/components/UserLabel/_variables.scss

```diff
@@ -0,0 +1,11 @@
+@use 'sass:map';
+
+$sizes: (
+    'xs': 24px,
+    's': 28px,
+    'm': 32px,
+    'l': 42px,
+    'xl': 50px,
+);
+
+$default-size: map.get($sizes, 'm');
```

Comment:
We can refactor styles to use size of `Avatar` component as a basis in layout and not duplicate these heights.